website/blog/2024-06-26-llm-as-judge/index.md

```diff
@@ -0,0 +1,528 @@
+            input="It's raining cats and dogs.",
+            output="Está lloviendo a cántaros.",
+            score=5,
+            justification="This translation uses a Spanish idiom that accurately conveys the meaning of heavy rain."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Kick the bucket.",
+            output="Patear el balde.",
+            score=1,
+            justification="This literal translation fails to convey the idiomatic meaning of dying."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Kick the bucket.",
+            output="Estirar la pata.",
+            score=5,
+            justification="This translation uses the equivalent Spanish idiom for dying, showing high cultural awareness."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Once in a blue moon.",
+            output="Una vez en una luna azul.",
+            score=2,
+            justification="This literal translation does not capture the rarity implied by the idiom."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="Once in a blue moon.",
+            output="De vez en cuando.",
+            score=4,
+            justification="This translation captures the infrequency but lacks the idiomatic color of the original."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="The ball is in your court.",
+            output="La pelota está en tu cancha.",
+            score=3,
+            justification="This translation is understandable but somewhat lacks the idiomatic nuance of making a decision."
+        ),
+        mlflow.metrics.genai.EvaluationExample(
+            input="The ball is in your court.",
+            output="Te toca a ti.",
+            score=5,
+            justification="This translation accurately conveys the idiomatic meaning of it being someone else's turn to act."
+        )
+    ],
+    model="openai:/gpt-4",
+    parameters={"temperature": 0.0},
+)
+```
+
+### The Toxicity Metric
+
+On top of this let's use MLflow default metrics for the evaluators. In this case MLflow wll use roberta-hate-speech model to detect the [toxicity](https://huggingface.co/spaces/evaluate-measurement/toxicity). This metric evaluates responses for any harmful or inappropriate content, reinforcing the company's commitment to a positive customer experience.
+
+```python
+# Log and evaluate the model
+with mlflow.start_run() as run:
+    results = mlflow.evaluate(
+        basic_translation_model.model_uri,
+        data=eval_data,
+        model_type="text",
+        evaluators="default",
+        extra_metrics=[cultural_sensitivity],
+        evaluator_config={
+        "col_mapping": {
+            "inputs": "llm_inputs",
+           }}
+   )
+    
+mlflow.end_run()
+```
+
+You can retrieve the final results as such:
+
+```python
+results.tables["eval_results_table"]
+```
+
+<style scoped>
```

Comment:
This doesn't compile. Can you check locally that CSS overrides adhere to the mdx syntax format?